src/main/resources/META-INF/plugin.xml

```diff
@@ -5,7 +5,7 @@
             url="https://research.cs.aalto.fi/LeTech/">LeTech Group at Aalto University
     </vendor>
     <!--  this is the range of supported IntelliJ IDEA version the plugin can be used with  -->
-    <idea-version since-build="213.5744.223"/>
+    <idea-version since-build="222.2680.4"/>
```

Comment:
I'd like to see a bit newer version here but it is what it is. 🤷‍♂️ Is this version installed in Aalto computers, do you know?